packages/nimble-components/src/utilities/tests/timeout.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Timeout for use in async tets.
+ */
+export async function timeout(ms = 0): Promise<void> {
+    await new Promise((resolve, _reject) => {
+        window.setTimeout(() => {
+            // eslint-disable-next-line no-void
+            resolve(void 0);
+        }, ms);
+    });
```

Comment:
Can we just do `resolve()`? I don't see why we need to pass `undefined`. If not for some weird reason can we pass `undefined` instead of `void 0`? That's a weird old hack we haven't needed since like ES3

```suggestion
    await new Promise((resolve) => {
        window.setTimeout(() => resolve(), ms);
    });
```

Reply:
Btw...you have to pass `undefined`.